draft-ietf-mls-architecture.md

```diff
@@ -943,7 +955,7 @@ message to a group which appears to be from any valid client since
 they cannot forge the signature.
 
 The MLS protocol will ensure that an adversary cannot compute any
-previous AEAD keys for the same epoch, or any other epochs.  Because
+previous or future AEAD keys for the same epoch, or any other epochs.  Because
```

Comment:
I'm not sure this is correct. After a full state compromise, the adversary can compute future AEAD keys for that epoch. We're not getting intra-epoch PCS with MLS. To make that work, we'd have to KEM an additional secret to the other group members with every application message. But that would make out-of-order messages impossible and application messages generally less efficient.

Reply:
I agree, will undo that change.